__tests__/__packages__/cli-test-utils/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe CLI test utils package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Refactored code to reduce the use of deprecated functions to prepare for Node 22 support. [#2191](https://github.com/zowe/zowe-cli/issues/2191)
```

Comment:
Can we explain why we need Node 22 support?

Reply:
I don't quite know how to explain it in the changelog. The CLI supports running on all Node LTS versions that are actively supported by the NodeJS project maintainers, and Node 22 will reach that point in October. That commitment is documented in the Zowe CLI system requirements. This PR sets the ground work to honor that commitment.

Reply:
How about: " ... upcoming NodeJS 22 support"

Reply:
Sounds good. Changelogs updated. Thanks!